README.md

```diff
@@ -304,6 +301,35 @@ It will also exclude any files with the extension `.spec.js`.
 **Note:** Be wary of automatic OS glob expansion when specifying include/exclude globs with the CLI.
 To prevent this, wrap each glob in single quotes.
 
+### Including files within `node_modules`
+
+We always add `**/node_modules/**` to the exclude list, even if not specified in the config.
+You can override this by setting `--exclude-node-modules=false`.
+
+For example, the following config will instrument everything in `lib/` and `node_modules/@my-org/`, except for `node_modules/@my-org/something/unwanted.js`.
```

Comment:
except for `node_modules/@my-org/something/unwanted.js` and files in any folder named `test`.  The idea of the `**/test/**` exclude is that it will exclude all `test` folders within `lib/` or `node_modules/@my-org/`.

Reply:
Yeah, I don't think I explained that very well, I think I was trying to highlight differences in behaviour and ignore the obvious stuff.  But that really isn't good enough for documentation.